src/lib/style/components/_cards.scss

```diff
@@ -15,19 +15,80 @@ $text-color-lighter: #898989;
   font-size: $text-size-base;
 }
 
-// TODO: Simplify and remove this scss, when we move to a single
-// card type there should be no special casing as is below
 .SRC-portalCard {
-  display: block;
-  color: $text-color-base;
+  display: block;  
   font-family: 'Lato', sans-serif;
   font-size: $text-size-base;
   line-height: $baseline-grid;
 
   border: 1px solid $color-divider;
-  margin-top: 30px; // just for test page
-  margin-bottom: 30px; // just for test page
+  margin-top: 30px;
+  margin-bottom: 30px;
+  
+  // default (non-header styles)
+  color: $text-color-base;
+  .SRC-type,
+  .SRC-cardThumbnail {
+    color: $text-color-lighter;
+  }
+  .SRC-cardThumbnail {
+    width: 15%;
+    img {
+      width: 50%;
+      &.iconImg {
+        max-width: 75px;
+      }
+    }  
+  }
+  .SRC-cardMetadata {
+    background-color: $color-gray-background;    
+  }
 
+  &.SRC-portalCardHeader {
+    color: white;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    
+    @media (min-width: map-get(SRC.$breakpoints, 'medium')) {
+      padding-top: 25px;
+    }
+    @media (max-width: map-get(SRC.$breakpoints, 'medium')) {
+      padding-top: 89px;
```

Comment:
Noticed a bug that's currently on staging.  When in a smaller viewport, the top of the header card is hidden behind the top nav.  Now increase the padding-top on mobile.